index.html

```diff
@@ -117,37 +117,30 @@ <h2>Network requests</h2>
 
       <p>
       A <dfn data-lt="network requests">network request</dfn> occurs when the
-      user agent must use the network to service a single <dfn data-cite="RFC7230#section-2.1">request</dfn>.
+      user agent must use the network to service a single [=request=].
       </p>
 
       <p>
-      If the user agent can service a <a>request</a> out of a local
-        cache, that <a>request</a> MUST NOT result in a <a>network
-        request</a>.
+      If the user agent can service a [=request=] out of a local cache, that
+      [=request=] MUST NOT result in a <a>network request</a>.
       </p>
 
       <p>
-      If the user agent follows [=redirect status|redirects=] as part of a [=navigate|navigation=],
-      there MUST be separate <a>network requests</a> for each <a>request</a> in
-      the redirect chain.
+      If the user agent follows [=redirect status|redirects=] as part of a
+      [=navigate|navigation=], there MUST be separate <a>network requests</a>
+      for each <a>request</a> in the redirect chain.
       </p>
 
       <p>
-      A <a>request</a> MUST NOT result in a <a>network request</a> if the user
-      agent is known to be offline (i.e., when <code>navigator.</code>{{NavigatorOnLine/onLine}} returns
-      <code>false</code>).
+      A [=request=] MUST NOT result in a <a>network request</a> if the user
+      agent is known to be offline (i.e., when <code>navigator.</code>
+      {{NavigatorOnLine/onLine}} returns <code>false</code>).
       </p>
 
       <p>
-      A <a>request</a> MUST NOT result in a <a>network request</a> if it is
-      blocked due to mixed content or CORS failures.  Any <a>CORS-preflight
-        request</a> MUST result in its own <a>network request</a>.
-      </p>
-
-      <p class="note">
-      For user agents that service <a>requests</a> according to the [[FETCH]]

```

Comment:
I took this note out because I'm trying to only use Fetch... but we can add it back or edit it if useful. If we are aware of other non-Fetch/non-browser based implementations, perhaps an edit is useful.